Documentation/cilium_operator.rst

```diff
@@ -0,0 +1,219 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+Cilium Operator
+===============
+
+This document provides a technical overview of the Cilium operator and describes
```

Comment:
The Operator is being referenced by subtly different names throughtout the doc, `cilium-operator`, `Cilium operator`, `operator`, etc. I suggest we stick to one: `Cilium Operator` and `Operator`.

Reply:
Ah, I think I still missed a few instances of this :man_facepalming: 
I will update the doc to make this reference consistent throughout. Thanks for pointing this out.